app/src/main/java/com/example/tasky/agenda/presentation/agenda_detail/AgendaDetailState.kt

```diff
@@ -5,39 +5,52 @@ package com.example.tasky.agenda.presentation.agenda_detail
 import androidx.compose.material3.ExperimentalMaterial3Api
 import com.example.tasky.agenda.domain.model.Attendee
 import com.example.tasky.agenda.domain.model.Photo
+import com.example.tasky.agenda.presentation.util.AgendaDetailBottomSheetType
 import com.example.tasky.agenda.presentation.util.AgendaItemAttendeesStatus
 import com.example.tasky.agenda.presentation.util.AgendaItemInterval
 import com.example.tasky.agenda.presentation.util.defaultAgendaItemIntervals
+import com.example.tasky.core.domain.ValidationItem
 import java.time.ZoneId
 import java.time.ZonedDateTime
 import java.time.temporal.ChronoUnit
 
 data class AgendaDetailState(
     val loadingInitialData: Boolean = false,
+    val isOnline: Boolean = false,
     val title: String = "Project X",
     val description: String = "Weekly plan\n Role distribution",
     val selectedAgendaReminderInterval: AgendaItemInterval = defaultAgendaItemIntervals().first(),
     val fromTime: ZonedDateTime = ZonedDateTime.now(ZoneId.of("UTC")).truncatedTo(ChronoUnit.HOURS),
     val selectedAttendeeStatus: AgendaItemAttendeesStatus = AgendaItemAttendeesStatus.ALL,
     val details: AgendaItemDetails? = AgendaItemDetails.Event(),
-    val imageLoading: Boolean = false
+    val agendaDetailBottomSheetType: AgendaDetailBottomSheetType = AgendaDetailBottomSheetType.NONE
 ) {
     val localFromTime: ZonedDateTime
```

Comment:
Suggestion: You could simplify this by not storing both `fromTime` and `localFromTime` inside the state. Keep only the canonical value (`fromTime` in UTC), and then convert it on demand when you actually need the local representation. That way you avoid managing two sources of truth, and the conversion stays a lightweight extension function:

```kotlin
fun ZonedDateTime.toLocal(): ZonedDateTime =
    withZoneSameInstant(ZoneId.systemDefault())

// usage
val localFromTime = state.fromTime.toLocal()
```